app/Actions/Diagnostics/Pipes/Checks/AppUrlMatchCheck.php

```diff
@@ -50,7 +61,7 @@ public function handle(array &$data, \Closure $next): array
 			$data[] = 'Error: LYCHEE_UPLOADS_URL is set and empty. This will prevent images to be displayed. Remove the line from your .env';
 		}
 
-		if (($config_url . '/uploads/') === $config_url_imgage && !$this->checkUrlMatchCurrentHost()) {
+		if (($config_url . ($dir_url !== '' ? $dir_url . '/' : '') . '/uploads/') === $config_url_imgage && !$this->checkUrlMatchCurrentHost()) {
```

Comment:
```suggestion
		if (($config_url . ($dir_url !== '' ? $dir_url : '') . '/uploads/') === $config_url_imgage && !$this->checkUrlMatchCurrentHost()) {
```
This would have resulted in a double slash?